src/Clue/GraphComposer/Command/Show.php

```diff
@@ -17,13 +17,14 @@ protected function configure()
              ->setDescription('Show dependency graph image for given project directory')
              ->addArgument('dir', InputArgument::OPTIONAL, 'Path to project directory to scan', '.')
              ->addOption('format', null, InputOption::VALUE_REQUIRED, 'Image format (svg, png, jpeg)', 'svg')
-           /*->addOption('dev', null, InputOption::VALUE_NONE, 'If set, Whether require-dev dependencies should be shown') */;
+             ->addOption('no-dev', null, InputOption::VALUE_NONE, 'Hide dev dependencies.');
```

Comment:
Shouldn't `no-dev` be replaced by `dev` as in the `Export` command?
It would keep consistency between the two commands, therefore ensure better understandability